sklearn/ensemble/_hist_gradient_boosting/tests/test_warm_start.py

```diff
@@ -169,22 +169,16 @@ def _get_rng(rng_type):
     gb_1 = GradientBoosting(n_iter_no_change=5, max_iter=2,
                             random_state=random_state)
     gb_1.fit(X, y)
-    train_val_seed_1 = gb_1._train_val_split_seed
-    small_trainset_seed_1 = gb_1._small_trainset_seed
+    random_seed_1 = gb_1._random_seed
```

Comment:
for completeness let's refit `gb_1` and assert that the seed is different here

Reply:
There is still a small chance (2 ** -32) that the seeds are equal. Let's hope that it never happens :)

Reply:
Actually never mind I think it's expected that the seed will be equal anyway since we pass an int and a fixed instance as `random_state`. I guess we should pass `None` instead?